qemu/tests/set_link.py

```diff
@@ -195,6 +195,8 @@ def set_link_test(linkid, link_up, expect_status,
     change_queues = False
     guest_ifname = ""
     guest_ip = vm.get_address()
+    host_ip = utils_net.get_ip_address_by_interface(params.get("netdst"))
```

Comment:
I have seen srchost defined in tests-example.cfg which represents the host ip address, and use function get_ip_address_by_interface() to make sure we can get host ip here. The second step become unnecessary actually. Will correct it as the comment you added.